src/main/java/com/google/sps/servlets/ClusterServlet.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+import java.util.ArrayList;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/api/v1/cluster")
+public class ClusterServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    ArrayList<TopChartsCluster> clusters = new ArrayList<TopChartsCluster>();
+
+    TopChartsCluster topCharts = new TopChartsCluster();
+    clusters.add(topCharts);
+
+    String jsonClusters = convertToJson(clusters);
+    response.setContentType("application/json;");
+    response.getWriter().println(jsonClusters);
+
+  }
+
+  private String convertToJson(ArrayList<TopChartsCluster> list) {
+    Gson gson = new Gson();
+    String json = gson.toJson(list);
```

Comment:
This will create a response like: `[{cluster},{cluster}]`.

Let's add a key to it, like `{ stream: [{cluster},{cluster}] }`.

This will make it easier to add new information to this API without breaking the client (i.e. frontend).